src/prefect/server/database/_migrations/versions/sqlite/2025_07_14_093336_385812ff8d32_add_log_flow_run_id_and_task_run_id_fk.py

```diff
@@ -0,0 +1,47 @@
+"""add_log_flow_run_id_and_task_run_id_fk
+
+Revision ID: 385812ff8d32
+Revises: 8bb517bae6f9
+Create Date: 2025-07-14 09:33:36.223352
+
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "385812ff8d32"
+down_revision = "8bb517bae6f9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # add foreign keys
```

Comment:
For some reason sqlite just drops all rows from the log so we didn't need the same delete where clause here.
I even tried adding it but sqlite still dropped everything :man_shrugging: